haralyzer/assets.py

```diff
@@ -12,10 +12,9 @@
 # I know this import is stupid, but I cannot use dateutil.parser without it
 from dateutil import parser
 
-assert parser
-
 from .compat import iteritems
 from .errors import PageNotFoundError
+from .sub_classes import Request, Response
```

Comment:
Not sure that I like the `sub_classes` namespace, it feels a bit too generic. Of course I am also blanking on a better name right now. The classes relate to modeling the requests/responses, so maybe something like `http` or `transport`?

Reply:
Will rename to http